commands/skills/skills.py

```diff
@@ -0,0 +1,9 @@
+from evennia.commands.cmdset import CmdSet
+from commands.skills.appraise import CmdAppraise
+
+class SkillCmdSet(CmdSet):
```

Comment:
I'm thinking since this is likely to be the only thing in this file, we may want to consider moving the `SkillCmdSet` definition into commands/skills/__init__.py. This would simplify imports in default_cmdsets.py